.travis.yml

```diff
@@ -10,9 +10,9 @@ script:
 - go test -v ./...
 - >
   echo "Checking goimports..." && (
-    if [[ -n $(goimports -l internal cmd) ]]; then
-      goimports -d internal cmd
-      echo 'Please run `goimports -w cmd internal`.'
+    if [[ -n $(goimports -l cmd internal pkg) ]]; then
+      goimports cmd internal pkg
+      echo 'Please run `goimports cmd internal pkg`.'
```

Comment:
Was the drop of the flags intentional?  `-d` provides the diff in the failing Jenkins terminal, and `-w` writes the changes in-place rather than dumping the entire codebase to the terminal when run locally.

Reply:
Yikes. Good catch, sorry about that.